app/serializers/event_serializer.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class EventSerializer
+  include JSONAPI::Serializer
+  attributes :id
+  attribute :eventTitle, &:title
```

Comment:
You should not change title key name, just leave title. The same for description

Reply:
Serialized only date:
```
  attributes :id, :title, :description
  attribute :eventDate do |object|
    object.date.to_fs(:short)

```